atc/metric/emitter/influxdb_client.go

```diff
@@ -0,0 +1,34 @@
+package emitter
+
+import (
+	"time"
+
+	influxclient "github.com/influxdata/influxdb1-client/v2"
+)
+
+// This is a copy of the github.com/influxdata/influxdb1-client/v2/client.Client interface whose sole purpose is
```

Comment:
I only added this file as a last resort. As stated in the counterfeiter docs for third party packages, I tried with
 //go:generate counterfeiter github.com/influxdata/influxdb1-client/v2/client.Client
and various other permutations of that, but none of them worked.